packages/plugin-trpc/cli/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+	"useTabs": true,
```

Comment:
Love it, this is the right setting :). What about the other settings in this file, do we need them or can we let Prettier use the default values?

I'm asking because each decision we make here may be not something developers who use this project want.

Reply:
If I'm not wrong these are the prettier settings sveltekit uses when creating a new sveltekit project 🤔 We can be less opinionated here though